lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -83,7 +90,8 @@ defmodule RealtimeWeb.RealtimeChannel do
         presence_key: presence_key(params),
         self_broadcast: !!params["config"]["broadcast"]["self"],
         tenant_topic: tenant_topic,
-        channel_name: sub_topic
+        channel_name: sub_topic,
+        public: !!public
```

Comment:
`is_public` already guarantees it is a boolean, so perhaps: `public: public` is enough?